src/v2/Components/__tests__/LotTimer.jest.tsx

```diff
@@ -96,13 +97,60 @@ describe("extendedBiddingInfoCopy", () => {
         "*Closure times may be extended to accomodate last minute bids"
       )
     })
+    describe("a bid has enxtended the auction", () => {
+      it("shows the extended next to the timer", () => {
+        let startDate = new Date()
+        const startAt = new Date(startDate.setMonth(startDate.getMonth() - 1))
+        let endDate = new Date()
+        const saleArtwork: LotTimer_saleArtwork = {
+          endAt: new Date(
+            endDate.setMinutes(endDate.getMinutes() + 1)
+          ).toISOString(),
+          formattedStartDateTime: "",
+          extendedBiddingEndAt: new Date(
+            endDate.setMinutes(endDate.getMinutes() + 3)
+          ).toISOString(),
+          sale: {
+            startAt: startAt.toISOString(),
+            extendedBiddingPeriodMinutes: 2,
+          },
+          " $refType": "LotTimer_saleArtwork",
+        }
+        const wrapper = mount(<LotTimer saleArtwork={saleArtwork} />)
+        const text = wrapper.text()
+        expect(text).toContain("Extended:")
```

Comment:
This is good, but could be extended to show the full timer string as well ("1m 23s" or whatever).

Reply:
How come this is resolved? It still just tests `Extended:`